lib/msal-core/src/messaging/MessageHelper.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { WindowUtils } from "./../utils/WindowUtils";
+import { MessageCache } from "./MessageCache";
+import { Logger } from "./../Logger";
+import { MessageDispatcher } from "./MessageDispatcher";
+
+export enum MessageType {
+    REDIRECT_REQUEST = "REDIRECT_REQUEST",
+    HASH = "HASH",
+    URL_TOP_FRAME = "URL_TOP_FRAME",
+    URL_NAVIGATE = "URL_NAVIGATE"
+};
+
+export enum WindowType {
+    TOP_FRAME,
+    IFRAME
+};
+
+export type PAYLOAD = {
```

Comment:
Does this need to be all caps?

Reply:
Need not be. Changed.

Reply:
Also I guess Daryl's initial suggestion was to name the data payload, not the entire message. Reflecting that comment too.